docs/proposals/scheduler-dispatcher.md

```diff
@@ -0,0 +1,116 @@
+#scheduler dispatcher proposal
+
+**Status**: Design & Implementation in progress.
+> Contact @combk8s or @mqliang for questions & suggestions.
+
+###Motivation
+
+In current Kubernetes design, there is only one default scheduler in a Kubernetes cluster, and the default scheduler must schedule
+pod sequentially, since it should take all the nodes into consideration and make an optimal decision, schedule concurrently is racy.
+If a lot of users create a majority of pods in a short time, a serially running scheduler would become the bottleneck of the whole
+cluster.  
+
+A multiple schedulers framework is undoutedly much more helpful, several Multi-Scheduler and Scheduler Extension proposals have been 
+came forward [#11793](https://github.com/kubernetes/kubernetes/issues/11793), [#13580](https://github.com/kubernetes/kubernetes/pull/13580). However, IMHO
+the Multi-Scheduler proposal has it's own drawbacks: scheduling conflicts is inevitable and hard to solve. Personally, letting Kubelet to
+solve the conflict problem is not a good solution, since it can make cluster into an uncertain state, especially when a lot of pods be scheduled concurrently, conflicts would happen very likely, and many pod would remain Pendding for a long time.
```

Comment:
Please site: kubernetes/kubernetes: Pull Request #17197

and imho you need to back this up with data, or some research that points to this.  There exists other scheduling systems that do very similar things to what was outlined in #17197.  Referenced in the Omega paper, condor does the same thing too.